src/javascript/app_2/pages/portfolio/portfolio.jsx

```diff
@@ -0,0 +1,272 @@
+import React from 'react';
+import moment from 'moment';
+import DAO from '../../data/dao';
+import DataTable from '../../components/elements/data_table.jsx';
+import Client from '../../../app/base/client';
```

Comment:
In order to keep them separated, please do not import anything from `app` into `app_2`. It's fine to import from `_common` folder.